pkg/operations/kubernetesupgrade/upgradecluster_test.go

```diff
@@ -291,10 +291,10 @@ var _ = Describe("Upgrade Kubernetes cluster tests", func() {
 		It("Should skip VMs that are already on desired version", func() {
 			mockClient.FakeListVirtualMachineScaleSetVMsResult = func() []compute.VirtualMachineScaleSetVM {
 				return []compute.VirtualMachineScaleSetVM{
-					mockClient.MakeFakeVirtualMachineScaleSetVM("Kubernetes:1.18.15"),
-					mockClient.MakeFakeVirtualMachineScaleSetVM("Kubernetes:1.18.10"),
-					mockClient.MakeFakeVirtualMachineScaleSetVM("Kubernetes:1.18.9"),
-					mockClient.MakeFakeVirtualMachineScaleSetVM("Kubernetes:1.18.15"),
+					mockClient.MakeFakeVirtualMachineScaleSetVM("Kubernetes:1.16.15"),
```

Comment:
just to prevent any unnecessary search/replace queries from coming up in the past, setting these UT data to a permanently deprecated version (the UT are just testing for version equality, they don't care about the supported versions list)